### ISSUE TITLE
rule proposal: enforce JSXClosingElement to be tag aligned

### ISSUE THREAD

**Issue by kentor** [yannickcr/eslint-plugin-react#912]
I don't believe there is currently a rule that checks that JSXClosingElements are aligned with the JSXOpeningElement.

In other words this should give a warning:

```jsx
<div>
  </div>
```

But this should not:
```jsx
<div>
</div>
```

**Comment by kentor** [comment]
ah never mind, `jsx-indent` works but didn't have an example
